### ISSUE TITLE
BUG: Tables appear twice in UI

### ISSUE THREAD

**Issue by ekreutz** [beekeeper-studio/beekeeper-studio#1564]
**Describe the bug**

For tables that have the same name _in different schemas_, the table name will appear twice in beekeeper's UI. The duplicates appear for all schemas that share table names. Using a Postgres 15 backend.

**To Reproduce**

1. Create two tables with the same name, in different schemas. A simple example might look like this:

```sql
CREATE SCHEMA schema1;

CREATE TABLE schema1.my_table (
  "id" INTEGER PRIMARY KEY
);

CREATE SCHEMA schema2;

CREATE TABLE schema2.my_table (
  "id" INTEGER PRIMARY KEY
);
```

2. Refresh and check the Beekeeper UI: the tables show up two times.

**Expected behavior**

Tables should show only once, per schema.

**Screenshots / Gifs**

![image](https://user-images.githubusercontent.com/9801173/230648380-a6703919-2a38-4563-9172-3ca631993380.png)

**Version Information (please complete the following information):**
 - OS: Ubuntu 22.04.2 LTS
 - App Version: 3.9.4
 - Database type and version: Postgres 15.2

**Additional context**

I often update Beekeeper using apt, so I'm not sure when this started happening. It feels like this is a very new issue that appeared in a version within the last 14 days or so.

**Comment by not-night-but** [comment]
@ekreutz Thanks for catching this! I've fixed the issue, it should get pushed out with the next release.
